torch/csrc/jit/script/module.h

```diff
@@ -184,6 +184,23 @@ struct TORCH_API Module {
     return module_object()->setAttr(name, v);
   }
 
+  void setattr(const std::string& name, IValue v) {
+    size_t slot = module_object()->type()->getAttributeSlot(name);
+    const TypePtr& expected = module_object()->type()->getAttribute(slot);
+    // TODO: give better error message when subtyping check fails, and python_str
```

Comment:
@suo This is where I think we should give better error message when subtyping check fails, and `python_str` are just mangled names from the same nn.Module, we shouldn't expose the mangled names to the user.